static/app/components/onboardingWizard/onboardingCard.tsx

```diff
@@ -0,0 +1,137 @@
+  org,
+  projects: allProjects,
+}: {
+  org: Organization;
+  projects: Project[];
+}) {
+  if (!org?.experiments.TargetedOnboardingMultiSelectExperiment) {
+    return null;
+  }
+
+  const [onboardingState] = usePersistedOnboardingState();
+  if (!onboardingState) {
+    return null;
+  }
+  // Projects selected during onboarding but not received first event
+  const projects = onboardingState.selectedPlatforms
+    .map(platform => onboardingState.platformToProjectIdMap[platform])
+    .map(projectId => allProjects.find(p => p.slug === projectId))
+    .filter(project => project && !project.firstEvent) as Project[];
+  return projects.length > 0 ? (
+    <Fragment>
+      <Heading>{t('Setup Projects')}</Heading>
+      <TaskCard key="onboarding-continue-card">
+        <OnboardingTaskProjectList>
+          {projects.slice(0, MAX_PROJECT_COUNT).map(p => (
+            <OnboardingTaskProjectListItem
+              key={p.id}
+              to={`/onboarding/${org.slug}/setup-docs/?project_id=${p.id}`}
+              onClick={() => {
+                trackAdvancedAnalyticsEvent('growth.onboarding_quick_start_cta', {
+                  platform: p.platform,
+                  organization: org,
+                });
+              }}
+            >
+              <OnboardingTaskProjectListItemInner>
+                <PlatformIcon platform={p.platform || 'default'} />
+                {p.slug}
+                <PulsingIndicator />
+                <PulsingIndicatorText>{t('Waiting for event')}</PulsingIndicatorText>
+                <IconChevron direction="right" />
+              </OnboardingTaskProjectListItemInner>
+            </OnboardingTaskProjectListItem>
+          ))}
+          {projects.length > MAX_PROJECT_COUNT && (
+            <OnboardingTaskProjectListItem
+              to={`/onboarding/${org.slug}/setup-docs/`}
+              onClick={() => {
+                trackAdvancedAnalyticsEvent('growth.onboarding_quick_start_cta', {
+                  organization: org,
+                });
+              }}
+            >
+              <OnboardingTaskProjectListItemInner>
+                <IconEllipsis />
+                {tct('and [num] more', {num: projects.length - MAX_PROJECT_COUNT})}
+              </OnboardingTaskProjectListItemInner>
+            </OnboardingTaskProjectListItem>
+          )}
+        </OnboardingTaskProjectList>
+      </TaskCard>
+    </Fragment>
+  ) : null;
+}
+
+const Heading = styled(motion.div)`
+  display: flex;
+  color: ${p => p.theme.purple300};
+  font-size: ${p => p.theme.fontSizeExtraSmall};
+  text-transform: uppercase;
+  font-weight: 600;
+  line-height: 1;
+  margin-top: ${space(3)};
+`;
+
+const TaskCard = styled(motion(Card))`
+  position: relative;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+`;
+
+const OnboardingTaskProjectList = styled('div')`
+  display: flex;
+  flex-direction: column;
+`;
+
+const OnboardingTaskProjectListItemInner = styled('div')`
```

Comment:
Any way to get away without having a 'inner' thing?

Reply:
Not really, because you need an outer div for the padding and an inner div for the border. Don't think there's a way to do away with `inner` here.

Reply:
Why can't you use a margin then?

Reply:
Because then when you hover your mouse over it, the `background-color` won't be applied to the region

Reply:
Cool yeah I see.